reference-implementation/to-upstream-wpts/transform-streams/general.js

```diff
@@ -300,4 +300,36 @@ promise_test(() => {
   return readableStreamToArray(ts.readable);
 }, 'methods should not not have .apply() or .call() called');
 
+promise_test(t => {
+  const transformer = {
+    transform() {
+      transformer.transform = undefined;
+      throw new TypeError();
+    }
+  };
+  const ts = new TransformStream(transformer);
+  return Promise.all([
+    promise_rejects(t, new TypeError(), ts.writable.getWriter().write('a'), 'write() should throw'),
+    promise_rejects(t, new TypeError(), ts.readable.getReader().read(), 'read() should throw')
```

Comment:
It's a little unclear to me whether these throw because it's undefined or throw because you threw a TypeError. If you're testing that the code gets exercised, then maybe setting a boolean, or using the `error1` pattern, would be clearer.